### PR TITLE
UI: handle alert in function update_alert

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

**Comment by pd0wm** [comment]
For some reason the camera view doesn't start for me, but don't see an obvious reason. Let's rebase and see if the problem is still there.

**Comment by pd0wm** [comment]
Still doesn't show the camera view

**Comment by deanlee** [comment]
My test is ok, I really don’t understand what’s going on

File: selfdrive/ui/ui.cc [hunk 1]
```diff
@@ -202,6 +178,57 @@ static void update_sockets(UIState *s) {
   s->started = scene.thermal.getStarted() || scene.frontview;
 }
 
+static void update_alert(UIState *s) {
+  if (!s->started || s->scene.frontview) return;
+
+  UIScene &scene = s->scene;
+  const uint64_t cs_frame = s->sm->rcv_frame("controlsState");
+  if (cs_frame == s->sm->frame) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/ui.cc:186]: use SubMaster's updated function

File: selfdrive/ui/ui.cc [hunk 2]
```diff
@@ -202,6 +178,57 @@ static void update_sockets(UIState *s) {
   s->started = scene.thermal.getStarted() || scene.frontview;
 }
 
+static void update_alert(UIState *s) {
+  if (!s->started || s->scene.frontview) return;
```
Comment (adeebshihadeh) [inline selfdrive/ui/ui.cc:182]: why does this check frontview? controlsState shouldn't be updated anyway

**Comment by adeebshihadeh** [comment]
I'm also not seeing the camera view.

File: selfdrive/ui/ui.cc (hunk 2 above)
Comment (deanlee) [reply selfdrive/ui/ui.cc:182]: checking the control times also requires this condition

**Comment by deanlee** [comment]
Can you tell me which script you used to run the camera view?

**Pull request merged**